src/App.js

```diff
@@ -1,24 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
```

Comment:
Removing this makes sense to me 👍 

I would also recommend deleting the `.svg` file from the repo, since it's no longer being used.